tests/testthat/test-get_risktable.R

```diff
@@ -363,10 +363,37 @@ testthat::test_that("T7.4 The output dataset has the attribute `statlist` that r
 
   risktable_bystat <- visR::get_risktable(survfit_object, group = "statlist")
   risktable_bystrat <- visR::get_risktable(survfit_object, group = "strata")
-  
+
   testthat::expect_equal(attr(risktable_bystat, "statlist"), "n.risk")
   testthat::expect_equal(attr(risktable_bystrat, "statlist"), base::sub('.*=', '', names(survfit_object$strata)))
 })
 
 # END OF CODE ----------------------------------------------------------
 
+# adding test for `get_risktable.tidycuminc()`
```

Comment:
Can you follow our conventions? If you need help, let us know.

Reply:
I updated trying to make it look the same. if it's not right, can you walk me through it or make the update?

Reply:
@SHAESEN2 let me know how you would like to proceed